crates/std_detect/src/detect/arch/x86.rs

```diff
@@ -74,6 +74,7 @@ features! {
     /// * `"avx512bitalg"`
     /// * `"avx512bf16"`
     /// * `"avx512vp2intersect"`
+    /// * `"knc"`
```

Comment:
Remove this line if you're not adding `knc`.

Reply:
Oops. Fixed.